.vitepress/theme/components/playground/InlinePlayground.vue

```diff
@@ -316,3 +316,13 @@ onMounted(async() => {
   }
 }
 </style>
+
+<style lang="scss" scoped>
+.inline-playground {
+  @apply mt-8 mb-8;
+  @media (min-width: 1600px) {
+    width: 150%;
+    margin-left: -25%;
+  }
+}
+</style>
```

Comment:
Personally I don't want to mix different css preprocessor together.

Reply:
Normally I would agree, however, because windi is primarily used by sass users (based on issues I have from the nuxt repo), it seems like a good idea to have a repo doing production tests that it's working

I can revert it if you're really against it

Reply:
Actually have removed